src/Core/src/Handlers/ScrollView/ScrollViewHandler.Windows.cs

```diff
@@ -20,6 +20,16 @@ protected override ScrollViewer CreatePlatformView()
 			return new ScrollViewer();
 		}
 
+		internal static void MapInvalidateMeasure(IScrollViewHandler handler, IView view, object? args)
+		{
+			handler.PlatformView.InvalidateMeasure(view);
+
+			if (handler.PlatformView.Content is FrameworkElement content)
+			{
+				content.InvalidateMeasure();
```

Comment:
Two thoughts about that:
* That handler is for the scroll viewer, not the content
* Also, using the handler to trigger InvalidateMeasure should already be happening here:
    * https://github.com/dotnet/maui/blob/c953ffb9cfc802d8c3fcf7def67423aaf036d14b/src/Controls/src/Core/Layout.cs#L500C5-L500C5

Reply:
The new device test is failing on Android, so I'm back to debugging that to see if an Android-specific fix is needed. I thought the [original bug](https://github.com/dotnet/maui/issues/14377) didn't repro on Android, but for some reason the test does repro.

Reply:
So the original repro does work on Android. But if the <Grid> parent of the <ScrollView> is gone, then it does repro. That could be handled as a different bug, log it, and just fix this for Windows now since it's working. The test would need an `#if !ANDROID` around it.

Reply:
The Android repro is covered by this bug:

* https://github.com/dotnet/maui/issues/17676